app/src/components/MovieListPage/MovieListPage.js

```diff
@@ -0,0 +1,196 @@
+/* eslint-disable no-unused-vars */
+import React, { useState, useEffect, useDebugValue } from 'react';
+import axios from 'axios';
+
+// components
+import Header from '../Header/Header';
+import SearchInput from '../SearchInput/SearchInput';
+import GenreList from '../GenreList/GenreList';
+import SelectFilter from '../SelectFilter/SelectFilter';
+import MovieCard from '../MovieCard/MovieCard';
+import MovieDetails from '../MovieDetails/MovieDetails';
+
+// styles
+import './MovieListPage.scss';
+
+// mocked data
+import genreListData from '../../mock/genreData.json';
+import sortByOption from '../../mock/sortByOption.json';
+import listOfGenres from '../../mock/mockOfAllAvailableGenre.json';
+
+const client = axios.create({
+  baseURL: 'http://localhost:4000/movies'
+});
+
+const MovieListPage = () => {
+  const [genre, setSelectedGenre] = useState(genreListData[0]);
+  const [sortType, setSortType] = useState(sortByOption[0].value);
+  const [currentSearchValue, setCurrentSearchValue] = useState('');
+  const [movieList, setMovieList] = useState([]);
+  const [selectedMovie, setSelectedMovie] = useState({});
+  const [isMovieDetailsOpen, setMovieDetailsOpen] = useState(false);
+
```

Comment:
Please retest scenarios when user applies several filters in time. E.g. user filter by search value and change sorting type